src/Polly.Extensions/Polly.Extensions.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net7.0;net6.0;netstandard2.0;net472;net462</TargetFrameworks>
+    <TargetFrameworks>net8.0;net7.0;net6.0;netstandard2.0;net472;net462</TargetFrameworks>
```

Comment:
Based on #1534 

Do we really need to add net8.0 target if we are not using anything in it?

Reply:
I thought it made the TimeProvider usage better/best? Without looking at the diff (on phone atm), I think we're using it to pass it through to some new APIs in System for Task.Delay?